packages/hardhat-core/src/internal/core/vars/vars-manager.ts

```diff
@@ -0,0 +1,145 @@
+import fs from "fs-extra";
+import debug from "debug";
+import { HardhatError } from "../errors";
+import { ERRORS } from "../errors-list";
+
+interface Var {
+  value: string;
+}
+
+interface VarsFile {
+  _format: string; // Version of the json vars file
```

Comment:
why not just `_version`?

Reply:
In several places in our code I found _format instead of _version.
@fvictorio which one is the correct one?